### ISSUE TITLE
"cursor: false;" error

### ISSUE THREAD

**Issue by misterfresh** [twbs/bootstrap-sass#881]
In "assets/stylesheets/bootstrap/_variables.scss" Line 227 the variable "$cursor-disabled: not-allowed !default;" becomes after compilation "cursor: false;"
At the moment overriding the variable $cursor-disabled by setting "$cursor-disabled: default;" removes the error.

**Comment by cvrebert** [comment]
I am unable to reproduce with Sass 3.4.13.
How are you compiling bootstrap-sass?

**Comment by misterfresh** [comment]
I compiled using libsass, gulp-sass.

**Comment by Francismori7** [comment]
Got the same thing over here.

```sass
  &[disabled],
  fieldset[disabled] & {
    cursor: $cursor-disabled;
  }
```
```sass
.form-control[disabled], fieldset[disabled] .form-control {
    cursor: false; }
```

**Comment by cvrebert** [comment]
Confirmed with
sassc: 3.1.0
libsass: 3.0.3
sass2scss: 1.0.3

**Comment by cvrebert** [comment]
This is presumably due to https://github.com/sass/libsass/issues/873, which is fixed in libsass 3.2.

**Comment by cvrebert** [comment]
Confirmed that this is broken in node-sass@2.1.1 and fixed in node-sass@3.0.0-beta.5, which uses libsass v3.2.0-beta.5.
